src/main/java/com/linecorp/sample/login/infra/line/api/v2/response/IdToken.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.sample.login.infra.line.api.v2.response;
+
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+public final class IdToken {
+    private DecodedJWT jwt;
```

Comment:
@YichingChiang 
I think this class doesn't need DecodedJWT class.
DecodedJWT is not IdToken's field.
https://developers.line.me/ja/docs/line-login/web/integrate-line-login/#part-715703e07a7d3b32
So, Could you remove this field ?
And you can set constructor's argument to decoded value.